src/status_im/navigation/core.cljs

```diff
@@ -192,7 +192,7 @@
                                               :keycard-pairing]))]
     {:init-root-fx (if keycard-account? :multiaccounts-keycard :multiaccounts)}))
 
-(defonce rset-app-launched
+(defonce reset-app-launched
```

Comment:
it should be `reg` or `register`, also `rset-lazy-reg` should be `set-lazy-reg`

Reply:
Maybe we can leave as it is as it functions the same.